sample-compose/buildSrc/src/main/java/Versions.kt

```diff
@@ -25,10 +25,9 @@ object Versions {
 
     const val JAVAX_INJECT_VERSION = "1"
 
-    const val KOTLIN_REFLECT_VERSION = "1.7.20"
     const val KOTLIN_VERSION = "1.7.20"
     const val KOTLINX_COROUTINES_VERSION = "1.6.4"
-    const val KOVER_VERSION = "0.6.0"
+    const val KOVER_VERSION = "0.6.1"
```

Comment:
May I know why we have to upgrade the version for `kover` 🙏

Reply:
@Wadeewee No, we don't need it 👍 I will leave this version upgrading back for other tasks 👌

Reply:
Fixed in https://github.com/nimblehq/android-templates/pull/384/commits/bda2a6a742ae459b455588e700ca4d387716ec29